src/main/java/net/logstash/logback/stacktrace/ShortenedThrowableConverter.java

```diff
@@ -159,6 +168,15 @@
     @Override
     public void start() {
         parseOptions();
+        // instantiate stack hasher if need be
+        if(inlineHash) {
+            // use same exclusion patterns to compute hash
+            if(excludes == null || excludes.isEmpty()) {
+                stackHasher = new StackHasher();
+            } else {
+                stackHasher = new StackHasher(StackElementFilter.byPattern(excludes));
```

Comment:
Should the StackHasher take the excludes into consideration?  The excludes are mainly there to shorten log output.  It seems like for error correlation you would always want to include all stack elements.  Your thoughts?